app/search/query/tests/test_base.py

```diff
@@ -0,0 +1,81 @@
+from unittest import TestCase
+from unittest.mock import patch
+
+from nose.tools import eq_, ok_
+
+from app.search.query.base import BaseQueryEngine
+from app.tests.memcached import mock_memcached
+
+
+QUERY = 'loveland'
+MOCK_RESULTS = [
+    {
+        'abstract': 'Welcome to <b>Loveland</b> Ski Area Near to the...',
+        'title': '<b>Loveland</b> Ski Area :: Colorado Ski Snowboarding...',
+        'url': 'http://skiloveland.com/'
+    },
+    {
+        'abstract': 'Loveland Ski Area is a ski area in the western United...',
+        'title': 'Loveland Ski Area - Wikipedia, the free encyclopedia',
+        'url': 'https://en.wikipedia.org/wiki/Loveland_Ski_Area'
+    }
+]
+MOCK_RESULT_SANITIZED = {
+    'abstract': 'Welcome to Loveland Ski Area Near to the...',
+    'title': 'Loveland Ski Area :: Colorado Ski Snowboarding...',
+    'url': 'http://skiloveland.com/'
+}
+
+
+class TestBaseQueryEngine(TestCase):
```

Comment:
The default timeout for `requests` is equal to `socket._GLOBAL_DEFAULT_TIMEOUT`, and there are tests for that. My default is to not test things that external libraries have tests for.